torchkbnufft/mri/dcomp_calc.py

```diff
@@ -4,7 +4,7 @@
 import torch
 
 
-def calculate_radial_dcomp_pytorch(nufftob_forw, nufftob_back, ktraj):
+def calculate_radial_dcomp_pytorch(nufftob_forw, nufftob_back, ktraj, stacks=False):
```

Comment:
Shouldn't this default to True to maintain backwards compatibility?

Reply:
Done